packages/radix-ui-themes/src/components/text-field.css

```diff
@@ -48,22 +57,6 @@
  *                                                                                                 *
  ***************************************************************************************************/
 
-.rt-TextFieldRoot {
-  display: flex;
-  box-sizing: border-box;
-}
-.rt-TextFieldSlot {
-  flex-shrink: 0;
-  display: flex;
-  align-items: center;
-}
-.rt-TextFieldInput {
-  display: block;
-  box-sizing: border-box;
-  padding: 0;
-  width: 100%;
-}
-
```

Comment:
Another sneaky issue when Slot color wouldn't work because of the order + specificity
https://themes-playground-modulz.vercel.app/test-textfield